src/as_bincode.rs

```diff
@@ -0,0 +1,38 @@
+use super::*;
+
+use std::fmt;
+
+#[derive(Debug)]
+pub struct AsBincode<T>(T);
+
+impl<T> Value for AsBincode<T>
+where
+    T: serde::Serialize + serde::de::DeserializeOwned + Name + fmt::Debug + 'static,
+{
+    type SelfType<'a> = T;
+
+    type AsBytes<'a> = Vec<u8>;
+
+    fn fixed_width() -> Option<usize> {
+        None
+    }
+
+    fn from_bytes<'a>(data: &'a [u8]) -> Self::SelfType<'a>
+    where
+        Self: 'a,
+    {
+        bincode::deserialize(data).expect("bincode deserialization error")
+    }
+
+    fn as_bytes<'a, 'b: 'a>(value: &'a Self::SelfType<'b>) -> Self::AsBytes<'a>
+    where
+        Self: 'a,
+        Self: 'b,
+    {
+        bincode::serialize(value).expect("bincode serialization error")
```

Comment:
Just occurred to me that big-endian should be used as bincode default to LE https://github.com/bincode-org/bincode/blob/trunk/docs/spec.md#endian

Reply:
Hmmm... . redb seems to be using `from_le_bytes` in the code for u64 etc, I guess to avoid doing any swapping on most popular CPU architectures, but it seems odd, because be doing it BE, lexicographical ordering would just work... . Interesting.